style.css

```diff
@@ -390,20 +461,255 @@ input::placeholder {
   line-height: 24px;
 }
 
-.form .btn {
+.contact-btncontainer {
   display: flex;
-  justify-content: center;
-  align-items: center;
-  margin: 1rem 0;
-  padding: 6px 6px;
-  position: absolute;
-  letter-spacing: 0.03em;
-  height: 48px;
-  left: 48px;
-  border: none;
-  color: #6070ff;
-  background-color: white;
-  font-size: 17px;
-  line-height: 24px;
-  max-width: 142px;
+  margin: center;
+}
```

Comment:
[OPTIONAL]
- [ ] Kindly note that `center` is not a `margin` value, thus it does not work, so it is either to select another value or remove the whole property for a cleaner code.